widgetssdk/src/main/res/layout/file_preview_activity.xml

```diff
@@ -14,7 +14,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:background="@color/glia_black_color"
-        app:navigationContentDescription="@string/glia_top_app_bar_navigate_up_content_description"
+        app:navigationContentDescription="@string/android_app_bar_nav_up_accessibility"
```

Comment:
All content descriptions should be setup in code, not in xml

Reply:
Fixed 09aa49436d2920b5797bdab2d56eb1ad29c3cdd2
Please have a look 😉